### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by batesyuk**
Add more information to README about getting the project running, unit testing, and manually linting the code.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,40 @@
...
+### SwiftLint
+
+The application uses SwiftLint for lining swift code. Rules can be found [here](.swiftlint.yml)
+If you want swiftlint to highlight issues in Xcode, make sure you have it installed. You can do this using [Homebrew](https://brew.sh/)
+
+`brew install swiftlint`
+
+### Rubocop
+
+The application uses Rubocop for lining Ruby code used in CI builds.
```
Comment (1pretz1) [inline README.md:37]: lining -> linting

File: README.md [hunk 2]
```diff
@@ -1 +1,40 @@
...
+To run this before pushing, make sure you have rubocop installed. This can be done with `gem install rubocop` although the prefered method is to use [Bundler](https://bundler.io/) and running `bundle install`
+
+Once you have Rubocop installed, you can check the app with `rubocop -A`
```
Comment (1pretz1) [inline README.md:40]: I believe `-A` auto-fixes the issues so could either update the command to just `rubocop` or mention that in the comment. May also be good to prefix the command with `bundle exec` (if we're updating the above line to only suggest bundling)

File: README.md [hunk 3]
```diff
@@ -1 +1,40 @@
...
+
+The application uses SwiftLint for lining swift code. Rules can be found [here](.swiftlint.yml)
+If you want swiftlint to highlight issues in Xcode, make sure you have it installed. You can do this using [Homebrew](https://brew.sh/)
+
+`brew install swiftlint`
+
+### Rubocop
+
+The application uses Rubocop for lining Ruby code used in CI builds.
+To run this before pushing, make sure you have rubocop installed. This can be done with `gem install rubocop` although the prefered method is to use [Bundler](https://bundler.io/) and running `bundle install`
```
Comment (1pretz1) [inline README.md:38]: As we're using bundler in the app project, could just remove the `gem install` part and only instruct to bundle install

File: README.md (hunk 2 above)
Comment (batesyuk) [reply README.md:40]: correct, I will remove the -A

**Review by 1pretz1 — APPROVED** [review]

**Pull request merged**